### PR TITLE
pax_utils -> 1.3.4 and updated deps, add gnulib_git

### PR DESCRIPTION
**Pull request by satmandu**
- Update pax_utils, and update depends
- `libcap` -> 2.64
- `libseccomp` -> 2.5.4
- `linux_pam` -> 1.5.2
- `py3_pyelftools` -> 0.28
- Add `gnulib_git`

Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/<satmandu/chromebrew.git CREW_TESTING_BRANCH=pax  CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/gnulib_git.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gnulib_git/v0.1-d6a07b4_x86_64/gnulib_git-v0.1-d6a07b4-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: '14dc5ef4dae60f1d48ea49849f66cacda35c901f57cd401f780715517286acf6',
+     armv7l: '14dc5ef4dae60f1d48ea49849f66cacda35c901f57cd401f780715517286acf6',
+       i686: '26cd24849f625113a04995fcc4923710a3e54102368770e6678bcfdb3d3bc040',
+     x86_64: '3215f4e675b309772d8d4b334e8ec5de4a94a463054aa7262b85adb1ef9d359f'
+  })
+
+  def self.build
```
Comment (uberhacker) [inline packages/gnulib_git.rb:28]: patch instead of build?

File: packages/pax_utils.rb [hunk 2]
```diff
@@ -3,38 +3,61 @@
...
+       _GL_FUNCDECL_SYS (strndup, char *,
+                         (char const *__s, size_t __n)
+    PATCHEOF
+    File.write('gnulib.patch', @gnulib_patch)
+    system 'patch -Np1 -i gnulib.patch'
+  end
+
   def self.build
+    # system './autogen.sh'
     system "sed -i 's|/usr/bin/env python|/usr/bin/env python3|g' lddtree.py"
```
Comment (uberhacker) [inline packages/pax_utils.rb:54]: Move to patch section?

File: packages/gnulib_git.rb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    system "sed -i \"/^[ ]*gnulib_dir=/s,\\`[^\\`]*\\`,#{CREW_PREFIX}/share/gnulib-git,\" gnulib-tool"
+  end
+
+  def self.install
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/gnulib-git"
+    FileUtils.install %w[check-module gnulib-tool], "#{CREW_DEST_PREFIX}/share/gnulib-git/", mode: 0o755
+    @dirs = %w[build-aux lib m4 modules top]
+    @dirs.each do |dir|
+      FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/gnulib-git/#{dir}"
+      FileUtils.cp_r Dir.glob("#{dir}/*"), "#{CREW_DEST_PREFIX}/share/gnulib-git/#{dir}/"
```
Comment (uberhacker) [inline packages/gnulib_git.rb:38]: You wouldn't need the FileUtils.mkdir_p or Dir.glob with:
```
FileUtils.cp_r dir, "#{CREW_DEST_PREFIX}/share/gnulib-git"
```

Comment (uberhacker) [reply packages/gnulib_git.rb:38]: Actually, you wouldn't even need the loop with:
```
FileUtils.cp_r %w[build-aux lib m4 modules top], "#{CREW_DEST_PREFIX}/share/gnulib-git"
```

**Comment by satmandu** [comment]
changes made

File: packages/pax_utils.rb (hunk 2 above)
Comment (uberhacker) [reply packages/pax_utils.rb:54]: This doesn't look like it was moved?

File: packages/gnulib_git.rb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+     armv7l: '14dc5ef4dae60f1d48ea49849f66cacda35c901f57cd401f780715517286acf6',
+       i686: '26cd24849f625113a04995fcc4923710a3e54102368770e6678bcfdb3d3bc040',
+     x86_64: '3215f4e675b309772d8d4b334e8ec5de4a94a463054aa7262b85adb1ef9d359f'
+  })
+
+  def self.patch
+    system "sed -i \"/^[ ]*gnulib_dir=/s,\\`[^\\`]*\\`,#{CREW_PREFIX}/share/gnulib-git,\" gnulib-tool"
+  end
+
+  def self.build end
```
Comment (uberhacker) [inline packages/gnulib_git.rb:32]: This line is unnecessary?

File: packages/pax_utils.rb [hunk 5]
```diff
@@ -3,38 +3,61 @@
...
+       #else
+       # if __GNUC__ >= 11
+      +#undef strndup
+       /* For -Wmismatched-dealloc: Associate strndup with free or rpl_free.  */
+       _GL_FUNCDECL_SYS (strndup, char *,
+                         (char const *__s, size_t __n)
+    PATCHEOF
+    File.write('gnulib.patch', @gnulib_patch)
+    system 'patch -Np1 -i gnulib.patch'
+    # system './autogen.sh'
```
Comment (uberhacker) [inline packages/pax_utils.rb:50]: This can be moved to the build section?

File: packages/pax_utils.rb (hunk 2 above)
Comment (satmandu) [reply packages/pax_utils.rb:54]: I don't understand why it isn't showing up. See the relevant commit on the branch for this PR: https://github.com/satmandu/chromebrew/commit/7896acf06011040c15c7722e0cbca8ad2f3bf02e

Comment (uberhacker) [reply packages/pax_utils.rb:54]: I think you moved the commented line instead of the sed line.

Comment (satmandu) [reply packages/pax_utils.rb]: Got it. Made that change.

**Pull request merged**